Microsoft.Toolkit.Uwp/Helpers/BluetoothLEHelper/BluetoothLEHelper.cs

```diff
@@ -144,6 +149,7 @@ public void StartEnumeration()
             _deviceWatcher.Added += DeviceWatcher_Added;
             _deviceWatcher.Updated += DeviceWatcher_Updated;
             _deviceWatcher.Removed += DeviceWatcher_Removed;
+            _deviceWatcher.EnumerationCompleted += _deviceWatcher_EnumerationCompleted;
```

Comment:
The StopEnumeration function doesn't remove the _deviceWatcher_EnumerationCompleted method